packages/design-system/src/components/SimpleMenu/Menu.tsx

```diff
@@ -180,8 +182,19 @@ const MenuItem = ({ onSelect, disabled = false, isLink, ...props }: ItemProps) =
           <Typography>{props.children}</Typography>
         </OptionLink>
       ) : (
-        <OptionButton cursor="pointer" color="neutral800" background="transparent" borderStyle="none" {...props}>
-          <Typography>{props.children}</Typography>
+        <OptionButton
+          cursor="pointer"
+          color="neutral800"
+          background="transparent"
+          borderStyle="none"
+          gap={2}
+          {...props}
```

Comment:
Can you remove startIcon and endIcon from the props object to avoid passing them to the `OptionButton` component please, in the component function definition above => `({ onSelect, disabled = false, isLink, startIcon, endIcon, ...props }: ItemProps)`.